ingestion/functions/README.md

```diff
@@ -49,7 +49,23 @@ Serverless Application Model
 Python and executed on a version 3.8 runtime. See set up instructions and
 common commands, below.
 
-### One-time setup
+### Setup for folks without AWS access
+
+If you're a first-time contributor to the project and don't have access to the S3 bucket containing the service account keys, you can run the ingestion and parsing functions fully locally, in the `retrieval/valid_scheduled_event.json`, add this auth param to it:
```

Comment:
This is really convenient -- did you try running the `sam local invoke [...]` command without credentials for our project, too? I can't recall exactly what permissions are required for the local invocations, but if we can completely remove giving AWS credentials from the equation, that's super nice.

Reply:
ah good call, yep just did it now (after renaming my ~/.aws folder temporarily).
`sam build` works fine.
`sam local invoke` of the retrieval function works as expected as well (fetches ok, fails to upload to s3 currently until I work on the local storage for testing):

```
{"errorType":"S3UploadFailedError","errorMessage":"Failed to upload /tmp/content.csv to epid-sources-raw/5f4d0a0a650106002eb179b8/2020/08/31/1442/content.csv: An error occurred (InvalidAccessKeyId) when calling the CreateMultipartUpload operation: The AWS Access Key Id you provided does not exist in our records."
```

I'll work on that once I get the parsing functions to also use the local cookies.